File: src/code/js/day_13.js
'use strict';

var _input = '';
var _index = 0;
process.stdin.on('data', (data) => { _input += data; });
process.stdin.on('end', () => {
    _input = _input.split(new RegExp('\n'));
    main();    
});
function readLine() { return _input[_index++]; }

/**** Ignore above this line. ****/

class Book {
    constructor(title, author) {
        if (this.constructor === Book) {
            throw new TypeError('Do not attempt to directly instantiate an abstract class.'); 
        }
        else {
            this.title = title;
            this.author = author;
        }
    }
    
    display() {
        console.log('Implement the \'display\' method!')
    }
}

class MyBook extends Book{
    
    constructor(title, author,price) {
    super(title,author)
    this.price = price
    }
    
    display() {
        process.stdout.write(`Title: ${this.title}\nAuthor: ${this.author}\nPrice: ${this.price}`)
    }
}

File: src/code/java/day_13.java
import java.util.*;

class Book {
    protected String title;
    protected String author;
    
    Book(String title, String author){
        this.title = title;
        this.author = author;
    }
     
}

class MyBook extends Book{
    
    private int price;
    
    MyBook(String title, String author, int price){ 
        super(title,author);
        this.price = price; 
    }
    
    public void display(){
        System.out.println("Title: " + title + "\nAuthor: " + author + "\nPrice: " + price);
    }
}

public class Solution {

    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        String title = sc.nextLine();
        String author = sc.nextLine();
        int price = sc.nextInt();
        
        MyBook mb = new MyBook(title, author, price);
        mb.display();
    }
}
